### DISCUSSION TITLE
Ordered list with annotation or table in the middle

### DISCUSSION THREAD

**Discussion by ixodie** [squidfunk/mkdocs-material#7208] [Q&A]
Hello,

I have an ordered list of tasks, but in the middle of the list I need to display a table or admonition.  After the table I want to return to the proper number of the ordered list.  Is there a way to do this?

Thanks!

**Comment by alexvoss** [comment]
Unfortunately, there is [no way to add attributes](https://python-markdown.github.io/extensions/attr_list/#limitations) to an `ol` element using `attr_lists`, otherwise I would suggest that.

You could use HTML for these lists and use the `start`attribute, perhaps using [md_in_html](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown#markdown-in-html) to make sure you do not need to turn everything into HTML.
